t/t0095-bloom.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/sh
+
+test_description='test bloom.c'
+. ./test-lib.sh
+
+test_expect_success 'get bloom filters for commit with no changes' '
+	git init &&
+	git commit --allow-empty -m "c0" &&
+	cat >expect <<-\EOF &&
+	Filter:
+	Length:0
+	Data:
+	EOF
+	test-tool bloom get_filter_for_commit "$(git rev-parse HEAD)" >actual &&
+	test_cmp expect actual 
+'
+
+test_expect_success 'get bloom filter for commit with 10 changes' '
+	rm actual &&
+	rm expect &&
+	mkdir smallFolder &&
+	cd smallFolder &&
+	for i in {0..10}
+	do
+		echo $i >$i
+	done &&
+	git add . &&
+	git commit -m "commit with 10 changes" &&
+	cd .. &&
+	cat >expect <<-\EOF &&
+	Filter:
+	Length:4
+	Data:
+	1734203087452766296
+	2310347025454204418
+	9030967673556582544
+	5773704018195857957
+	EOF
+	test-tool bloom get_filter_for_commit "$(git rev-parse HEAD)" >actual &&
+	test_cmp expect actual
+'
+
+test_expect_success EXPENSIVE 'get bloom filter for commit with 512 changes' '
+	rm actual &&
+	rm expect &&
+	mkdir bigFolder &&
+	cd bigFolder &&
+	for i in {0..512}
+	do
+		echo $i >$i
+	done &&
+	git add . &&
+	git commit -m "commit with 513 changes" &&
+	cd .. &&
+	cat >expect <<-\EOF &&
+	Filter:
+	Length:0
+	Data:
+	EOF
+	test-tool bloom get_filter_for_commit "$(git rev-parse HEAD)" >actual &&
+	test_cmp expect actual
+'
+
+test_expect_success 'compute bloom key for empty string' '
+	cat >expect <<-\EOF &&
+	Hashes:
+	|1444249612|1536583008|1628916404|1721249800|1813583196|1905916592|1998249988
```

Comment:
I notice you concatenate the hashes in a line here, while the data strings below are one-per-line. Any reason for that?
Perhaps always use line splits for clarity.